src/api_server/Cargo.toml

```diff
@@ -18,8 +18,8 @@ actix-web-httpauth = "0.8.0"
 aes-gcm = "0.10.1"
 anyhow.workspace = true
 async-trait.workspace = true
-as-types = { git = "https://github.com/confidential-containers/attestation-service.git", rev = "9484f06"}
-attestation-service = { git = "https://github.com/confidential-containers/attestation-service.git", default-features = false, rev = "9484f06", optional = true}
+as-types = { git = "https://github.com/confidential-containers/attestation-service.git", rev = "c33ae40"}
+attestation-service = { git = "https://github.com/confidential-containers/attestation-service.git", default-features = false, rev = "c33ae40", optional = true}
```

Comment:
Now you can update the rev of AS to the newest version: maybe [70796ef](https://github.com/confidential-containers/attestation-service/commit/70796efff9a85cf9ac5f7a234e4b254f8cf3e6fa)